packages/venia-ui/lib/components/CommunicationsPage/communicationsPage.css

```
@@ -0,0 +1,50 @@
+.root {
+    padding: 20px;
```

Preference for rems over pixel values when possible:

```suggestion
    padding: 1.25rem;
```